src/spaceone/inventory/model/workload/pod/data.py

```diff
@@ -95,6 +95,7 @@ class Pod(Model):
     restarts = IntType(serialize_when_none=False)
     age = StringType(serialize_when_none=False)
     containers = StringType(serialize_when_none=False)
+    pod_logs = ModelType(ObjectMeta, serialize_when_none=False)
```

Comment:
I think it would be better to define a separate model class for pod_logs .

```python
class PodLogs(Model):
    name = StringType(serialize_when_none=False)
    namespace = StringType(serialize_when_none=False)
```


```python
  ...
  pod_logs = ModelType(PodLogs, serialize_when_none=False)
```